core/utils/gtp.h

```diff
@@ -24,8 +24,8 @@ namespace utils {
 			const uint8_t *pktptr = (const uint8_t *)this;
 			size_t len = sizeof(Gtpv1);
 
-			if (gtph->seq)
-				len += 2;
+			if (gtph->seq)			/* TODO: Sequence # len set to 4B; verify this! */
```

Comment:
Create an issue to track this and reference it

Reply:
Done.